### ISSUE TITLE
Enabling Network policies is breaking POD to SVC communication

### ISSUE THREAD

**Issue by vysr2939** [kubeovn/kube-ovn#3347]
# Expected Behavior
To use network policies seamlessly without breaking the cluster aka POD to SVC communication  
# Actual Behavior
When Network Policies is enabled using the flag --enable-np=true and after we create first network policy, communication b/w pods to SVC which includes POD to API server communication is broken.

 Observation so far is once we enable network policies kube-ovn create a port group called "node.$node_name" which has all the ports associated to all pods. This is potentially blocking the communication.

# Steps to Reproduce the Problem

1. to start with ensure kube-ovn has network policy disabled i.e "-enable-np=false" 
![image](https://github.com/kubeovn/kube-ovn/assets/13367004/bddd38c1-7976-4b1f-8421-27a23a7951ee)

2. Create Test pod to check POD to API server communication. Everything works fine
![image](https://github.com/kubeovn/kube-ovn/assets/13367004/c1b7b1a6-9b2b-4650-a128-43d2f858a271)

![image](https://github.com/kubeovn/kube-ovn/assets/13367004/79ff0025-cace-4865-a947-2261fe73b388)

3  Enable Network Policy  i.e "-enable-np=true"
![image](https://github.com/kubeovn/kube-ovn/assets/13367004/747a88f8-22f8-4869-a984-21286a727a0f)

4. Create one sample network policy 

![image](https://github.com/kubeovn/kube-ovn/assets/13367004/8b936c62-b157-475a-b8c9-3b09296c63f6)

6. Check the POD to kube API server connectivity. Its broken. All custom controllers trying to reach kube-apiserver crashes
![image](https://github.com/kubeovn/kube-ovn/assets/13367004/42df9c05-fa33-48ea-9677-fe4fbfa75b57)

7. Delete the port group called "node.$node_name". Check the connectivity it works fine. Also other controllers become healthy. After sometime controller reconciles and port group is created again and communication is blocked 

![image](https://github.com/kubeovn/kube-ovn/assets/13367004/eba8a2a6-ce06-4dc9-aab9-4d768451d3ad)

# Additional Info

- Kubernetes version: v1.24.0

  **Output of `kubectl version`:**

  ```
![image](https://github.com/kubeovn/kube-ovn/assets/13367004/05ba328c-bf76-4a06-a8a5-60108ceae495)

  ```

- kube-ovn version: v1.12.1
  
  ```
![image](https://github.com/kubeovn/kube-ovn/assets/13367004/15551069-9822-4575-bacf-7ff058e4fdd1)
  ```

- operation-system/kernel version:
  
  **Output of `awk -F '=' '/PRETTY_NAME/ { print $2 }' /etc/os-release`:**
  ```
"Ubuntu 20.04.6 LTS"
  ```

  **Output of `uname -r`:**
  ```
  5.4.0-144-generic
  ```

**Comment by nics90** [comment]
Even I am facing the same issue and it is a blocker for us, since due to this issue we are not able to proceed further with kube ovn version v1.12.1. 

@Team : Could you please help here.

**Comment by oilbeater** [comment]
@changluyi can you take a look at this?

**Comment by changluyi** [comment]
it's a bug, use kubectl ko trace , it hit the acl rule allow-stateless, and will not establish conntrack in kernel.

this bug may introduce in ovndb code refactor. the version before and including 1.11 does not contain this bug.

**Comment by Kovendan03** [comment]
@changluyi , I have faced the same issue , waiting for this fix as well. Could you please let us know if this fix will be available in 1.12.0, 1.12.1 and 1.12.2 as well.

**Comment by oilbeater** [comment]
@patriziobassi we will publish 1.12.3 this week to contain this fix
